tools/cobbler-access.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Script to generate Cobbler credentials
+
+tmpfile=$(mktemp)
+
+# Basically `mkpasswd` but uses a small subset of special characters
+password=$(head /dev/urandom | tr -dc 'A-Za-z0-9!@#$%&' | head -c 12 && echo)
```

Comment:
What about `pwgen`?

Reply:
Never heard of it.  Also not pre-installed on Fedora.

Seems silly to install a dependency for a script like this.  Just my opinion though.

Reply:
pwgen is like 16kb and is really useful, but this is fine too